### ISSUE TITLE
Incorrect no space warning.

### ISSUE THREAD

**Issue by joelbrunetti** [dperson/transmission#19]
I frequently experience an error: 'Error: No space left on device' followed by a torrent path.
The download is also paused.

There is plenty of space left on the device. Resuming the download will work but sometimes the same torrent will cause the same error multiple times.

Looking at the output from 'docker logs transmission I see:
[time stamp] Torrent name write failed for "file name": No space left on device (inout.c:136).

I didn't have this problem with transmission on Debian so I figure it is an interaction with docker.
Any ideas? Could I have something silly in my configs?

Thanks!

**Comment by titpetric** [comment]
Hi,

TL;DR: If you're into a quick fix, add a -v (volume) mount for the complete /var/lib/transmission-deamon folder, not just the downloads folder within it. You can at least mount the `incomplete` folder as well.

Why? Transmission by default includes 3 folders of importance: `info` to store torrent metadata, `incomplete` where it stores the file during download, and `downloads`, the folder where the file gets moved to, when the download completes.

The documentation by default suggests only to mount the `downloads` folder, which was my mistake as well. The space for partial downloads is mapped to a docker volume, which lives on `/var/lib/docker` or some similar path. For me, this counts as part of the `/` filesystem, while -v mounted another disk for the download path.

Unfortunately, this is impossible to debug with running for example `docker ps -a -s`, as it will not report this size anywhere. I think it has something to do with keeping the file handle of the running download open - running `df` will show true disk usage however, so you can verify if you're *actually* running out of disk space, albeit temporary.

**Comment by dperson** [comment]
Thanks for the suggestion, I've updated the `README.md` example.

As a side note, that's how I've been running the container myself with a volume for `/var/lib/transmission-daemon`. Probably why I hadn't see the issue myself.

**Comment by nelsonblaha** [comment]
Where would one add such a -v flag? /etc/fstab? Transmission's config?
